_data-prepper/pipelines/configuration/sinks/s3.md

```diff
@@ -76,11 +76,11 @@ Use the following options when customizing the `s3` sink.
 
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
-`bucket` | Yes | String | The name of the S3 bucket to which the sink writes.
-`codec` | Yes | [Codec](#codec) | The codec that determines how the data is serialized in the S3 object.
+`bucket` | Yes | String | The object from which the data is retrieved and then stored. The `name` must match the name of your object store.
```

Comment:
What was the motivation for this rewording? The value provided here is the name of a bucket for writing.

In S3 terminology, the bucket is where objects are stored. So the "bucket" term is more accurate this this location. Perhaps:

> The name of the S3 bucket to which objects are stored